compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/PackageFileWriter.java

```diff
@@ -35,9 +35,6 @@
     public static byte[] writePackage(PackageFile packageFile) throws IOException {
         ByteArrayOutputStream byteArrayOS = new ByteArrayOutputStream();
         try (DataOutputStream dataOutStream = new DataOutputStream(byteArrayOS)) {
-            dataOutStream.writeInt(PackageFile.MAGIC_VALUE);
```

Comment:
Shall we remove this `PackageFile.MAGIC_VALUE` & `PackageFile.LANG_VERSION` as they are not used in other places?
Can be done through another PR as well.

Reply:
Creatde https://github.com/ballerina-platform/ballerina-lang/issues/32972 seems we can remove few methods as well.